### DISCUSSION TITLE
Colemak DHm Layout Available?

### DISCUSSION THREAD

**Discussion by alexchaveztech** [manna-harbour/miryoku#136] [Q&A] [answered]
Does the feature exist to use ColemakDHm layout variant? Meaning the letter m and k are swapped resulting in the letter m in the home row and k in the bottom row. I did not find anything on this subject and it's a popular layout in Matrix style keyboards.

**Comment by alexchaveztech** [comment]
Upon further investigation I have learned that the default Colemak layout is actually the default alphas option and there is no need to change it. :)

**Comment by manna-harbour** [comment, marked as answer]
The Miryoku default alpha arrangement has always been Colemak Mod-DHm, but DHm has been renamed to DH. See https://colemakmods.github.io/mod-dh/#revision-history. Other Colemak flavours are also available. See https://github.com/manna-harbour/miryoku/tree/master/docs/reference#base, https://github.com/manna-harbour/miryoku/tree/master/docs/reference#alphas, and https://github.com/manna-harbour/miryoku_babel#alphas-1.
